_posts/2021-04-27-rust-dev-workflow.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "A New Development Workflow for Arrow's Rust Implementation"
+date: "2021-04-27 00:00:00"
+author: ruanpa
+categories: [application]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow Rust community is excited to announce that its migration to a new development workflow is now complete! If you're considering Rust as a language for working with columnar data, read on and see how your use case might benefit from our new and improved project setup.
+
+In recent months, members of the community have been working closely with Arrow's [Project Management Committee][1] and other contributors to expand the set of available workflows for Arrow implementations. The goal was to define a new development process that ultimately:
+- Enables a faster release cadence
```

Comment:
```suggestion
- Enables a faster release cadence and versioning scheme that better matches Cargo's SemVer expectations
```

Reply:
> I think it looks great. Thank you @returnString !
> 
> I left some suggestions that I am not 100% thrilled with -- what I was trying to do was to highlight the specific changes that we hope to make (namely using crate versioning more in line with semver, as well as centralizing on a single tool)
> 
> However, it makes the initial bullet points more verbose. I do not feel strongly about this.
> 
> All in all, this is really nice

I've taken the spirit of these suggestions and tried to incorporate them without affecting the brevity of the bullet points too significantly - SemVer was worth mentioning for sure!